### DISCUSSION TITLE
Prevent moving a vector layer

### DISCUSSION THREAD

**Discussion by Jipforester** [opengisch/QField#5428] [Q&A]
Good morning,

I work on the field with teams who have to take points from a vector layer. I would like to know if it is possible to have an adjustable parameter that prevents the movement of a vector entity because it is active directly after selecting an entity. I would like them to be able to modify attribute table settings but not be able to move the points. Or in another case, if there is a parameter which shows that an entity has been moved and we can configure this from the form, that would be interesting, especially with the new $x and $y coordinates. Has anyone faced this problem and found a solution?

Thanks,

Jean-Philippe

**Comment by nirvn** [comment]
@Jipforester , have you tried the 'lock geometry' option for a given vector layer:

![image](https://github.com/opengisch/QField/assets/1728657/aabe43c2-398b-432e-b36f-f10aa7aeb760)

You need the QGIS qfieldsync plugion to configure that. Sounds like what you are looking for.

**Reply by Jipforester** [reply]
Hello Nirvn,

I have tried your solution and yes it works, my points (trees) can not move ! 

Thank you very much !
